mungegithub/mungers/close-stale.go

```diff
@@ -222,15 +222,19 @@ func findLatestWarningComment(obj *github.MungeObject) (*githubapi.IssueComment,
 	return lastFoundComment, true
 }
 
-func durationToDays(duration time.Duration) string {
-	days := duration / day
+func dayPhrase(days time.Duration) string {
```

Comment:
I haven't reviewed this PR at all, I quickly glanced at it, but I just know that this is incorrect. 

A `time.Duration` can't be `days`. If you want to count days, please use another type (like `int`)